parm/config/gfs/config.resources

```diff
@@ -585,7 +585,7 @@ case ${step} in
         exit 4
     esac
 
-    walltime="00:10:00"
+    walltime="01:30:00"
```

Comment:
Same here!

Reply:
I think this is because of copying the 80 members.  I'll see if the data exists to see what recent timings are.  As a note, most of these are "this works" not "this is the most optimal configuration".

Reply:
I see.
I am concerned that raising these limits (and others in this PR that are targeted for C1152) for all cases and all resolutions will considerably slow down our ability to turn around CI tests.  The limitation is in the global-workflow's configuration and setup part.

Reply:
I understand that concern.  I think there is a need to support both running quick CI/testing and being able to run at our operational target resolutions.   Moving forward: Do we find proper "ifs" for the two settings within the set up or do we wait for a more comprehensive solution, and back out the resource changes and merge the rest?  Or do we just let GFSv17 continue to keep these types of things in a fork/branch?

Reply:
A comprehensive solution is not going to appear any time soon.  This is a known and documented risk in the project. 
We have to choose between the fork and maintain or introduce `if` and maintain. Both add to the debt.
I would vote for `if` since it at least keeps the code intact.
I don't know when we will start to pay down the debt (if any).

Reply:
@aerorahul  - There are only so many hours in a day and a lot of priorities to juggle.  I'll brainstorm some possible ifs for various scenarios and try that out and we can go from there. 

I'm assuming the major concern is for places where we do not have the C1152 case, correct?